src/screens/Playlist/Playlist.tsx

```diff
@@ -35,6 +35,7 @@ function Playlist({
 
   const categories = getFiltersFromConfig(config, id);
   const filteredPlaylist = useMemo(() => filterPlaylist(playlist, filter), [playlist, filter]);
+  const shouldShowFilter = !isLoading && !isPlaceholderData && Boolean(categories.length);
```

Comment:
Stylistically, it would be nice if we decide on either `Boolean` or `!!` and try to use it everywhere. Right now we have a big mix.